src/kaltura-player.js

```diff
@@ -40,13 +40,16 @@ class KalturaPlayer extends FakeEventTarget {
   constructor(options: KPOptionsObject) {
     super();
     this._eventManager = new EventManager();
-    this._localPlayer = loadPlayer(options);
+    const {sources} = options;
```

Comment:
what happens if `sources` doesn't exist? what is the value?

Reply:
we have default sources